### PR TITLE
#57 - systelab-spinner: Remove the fillUnitsWithZero property or make…

### PR DESCRIPTION
**Pull request by rserra91**
… it more generic

# PR Details

## Description

Now we can specify the number of 0 added.

## Related Issue

https://github.com/systelab/systelab-components/issues/57

## Motivation and Context

Improve the application

## How Has This Been Tested

Added more test case.

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] I have read the **CONTRIBUTING** document
- [x] My code follows the code style of this project
- [ ] My change requires a change to the documentation 
- [ ] I have updated the documentation accordingly (README.md for each UI component)
- [x] I have added tests to cover my changes (at least 1 spec for each UI component with the same coverage as the master branch)
- [x] All new and existing tests passed
- [ ] A new branch needs to be created from master to evolve previous versions
- [x] Increase version in package.json following [Semantic Versioning](https://semver.org/)
- [ ] All UI components must be added into the showcase (at least 1 component with the default settings)
- [ ] Add the issue into the right [project](https://github.com/systelab/systelab-components/projects) with the proper status (In progress)

### CODE REVIEW

File: projects/systelab-components/src/lib/spinner/spinner.component.spec.ts [hunk 1]
```diff
@@ -14,15 +14,16 @@ import { TouchspinComponent } from './spinner.component';
...
                     <label class="label-value">{{values.value}}</label>
                 </div>
 	          `,
 	styles:   []
 })
 export class SpinnerTestComponent {
 	public values = new TouchSpinValues(34, 1, 100);
-	
+	public fillUnitsWithZero: boolean | number =false;
+	public strValue: string;
```
Comment (olgapuig) [inline projects/systelab-components/src/lib/spinner/spinner.component.spec.ts:26]: Use the same variable name "valueStr" for consistency.

File: projects/systelab-components/src/lib/spinner/spinner.component.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ export class TouchspinComponent {
 	@Input() public disabled = false;
 	private _spinValues: TouchSpinValues;
 	protected _valueStr: string;
-	@Input() fillUnitsWithZero = false;
+	@Input() fillUnitsWithZero:boolean|number = false; //0 if false, and 1 if true
```
Comment (olgapuig) [inline projects/systelab-components/src/lib/spinner/spinner.component.ts:17]: Document the new Input in the README (https://github.com/systelab/systelab-components/tree/master/projects/systelab-components/src/lib/spinner)

**Review by olgapuig — CHANGES_REQUESTED** [review]
Thanks for the enhancement @rserra91!
The version in the package.json needs to be increased following semantic versioning.

File: projects/systelab-components/src/lib/spinner/spinner.component.spec.ts (hunk 1 above)
Comment (rserra91) [reply projects/systelab-components/src/lib/spinner/spinner.component.spec.ts]: done

File: projects/systelab-components/src/lib/spinner/spinner.component.ts (hunk 2 above)
Comment (rserra91) [reply projects/systelab-components/src/lib/spinner/spinner.component.ts:17]: done

**Review by olgapuig — APPROVED** [review]

**Review by VictorDeBlas — APPROVED** [review]

**Review by carlosra85 — APPROVED** [review]

**Review by olgapuig — APPROVED** [review]

**Pull request merged**